Base/usr/share/man/man6/Snake.md

```diff
@@ -11,5 +11,13 @@ $ Snake
 ```
 
 ## Description
+`Snake` is an arcade game in which you navigate Snake to eat food. With each item of food you eat Snake grows longer by one block. The challenge is to keep growing without colliding with yourself. Once this happens, the game is over.
```

Comment:
Seems like you're naming the snake in the game `Snake` with this update, but I'm all for it :) :snake:

Reply:
Yeah - in my mind Snake is the name of the game but also of the character. Thanks for the review!